### PR TITLE
Update meta tags

### LINKED ISSUE #49: Clean up the meta tags
* replace icons
* theme colour
* title
* description
* add open graph tags to please the gods of SEO

_Originally posted by @dariatsvetkova in https://github.com/the-collab-lab/tcl-22-smart-shopping-list/pull/48#discussion_r635464010_

### PR DESCRIPTION
**Pull request by dariatsvetkova**
_For an example of how to fill this template out, [see this Pull Request](https://github.com/the-collab-lab/tcl-3-smart-shopping-list/pull/44)._ 

## Description

This PR takes care of all the meta information in the app:
* icons
* theme colour
* title
* description
* open graph tags

I did some research into React Helmet and decided not to use it for this task. According to [this issue](https://github.com/nfl/react-helmet/issues/213), React Helmet can only inject basic meta-tags on the client-side. For open graph/schema/twitter tags the app would need to be rendered on the server-side.

The main purpose of this PR is to make the app look good when we share it on LinkedIn and Twitter, so basic meta-tags aren't enough. That's why I updated the tags manually instead (let me know if there is a better solution).

## Related Issue

Closes #49 

## Acceptance Criteria

* New favicon and page title are showing up in the browser
* Theme colour is green (as seen in mobile browser)
* New title and meta description are showing up when you inspect the `<head>` element and when you share the link on social media
* Custom preview is showing up when you share the link on social media (so far it is showing up in Facebook and messaging apps; Twitter and LinkedIn are supposed to be able to fetch images from the production site, once this PR is merged).

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|  ✓ | :sparkles: New feature     |

### Before

image

### After

image
image
Untitled
photo_2021-05-28_22-36-35

## Testing Steps / QA Criteria

Use the [preview link](https://tcl-22-shopping-list--pr65-dt-meta-tags-7c04n13h.web.app) to see how it behaves:

* in the browser (title, favicon, meta description)
* in a mobile browser (theme colour)
* when shared on social media (but maybe don't actually share it just yet)

You can also see how [this tool](https://www.opengraph.xyz/) parses the tags.

### CODE REVIEW

**Review by cafloyd — APPROVED** [review]
So beautiful! can't wait to share to my social media! 😁

File: public/manifest.json [hunk 1]
```diff
@@ -20,6 +20,6 @@
   ],
   "start_url": ".",
   "display": "standalone",
-  "theme_color": "#000000",
-  "background_color": "#ffffff"
+  "theme_color": "#5F9645",
```
Comment (cafloyd) [inline public/manifest.json:23]: it's such a small thing, but I love the effect on the browser that comes from adding a theme color!

File: public/index.html [hunk 2]
```diff
@@ -1,46 +1,48 @@
...
+
+    <meta property="og:image" content="%PUBLIC_URL%/og-img-lg.png" />
+    <meta property="og:image:type" content="image/jpeg" />
+    <meta property="og:image:width" content="600" />
+    <meta property="og:image:height" content="314" />
+    <meta property="og:image:alt" content="AvoCart logo" />
+
+    <meta name="twitter:card" content="summary">
+    <meta name="twitter:title" content="AvoCart - Your Smart Shopping List">
+    <meta name="twitter:description" content="Smart shopping list that learns your buying habits and helps you remember what you need to buy on your next trip to the store.">
```
Comment (kkatoch89) [inline public/index.html:36]: This is awesome, thanks @dariatsvetkova

**Review by kkatoch89 — APPROVED** [review]
Is it just me or does getting the favicon feel like the moment when the app feels legit? LGTM!

**Review by ablizben — APPROVED** [review]
This looks wonderful! Thank you, Daria!

**Review by jonathanyeong — APPROVED** [review]
Woohoo great work on filling out all the meta tags! This is awesome.

**Pull request merged**